website/_sass/custom/custom.scss

```diff
@@ -0,0 +1,24 @@
+// We should override $nav-width(-md), however this breaks code highlighting and other styles.
+// This seems to be an issue for following the recommended approach.
+// See https://github.com/just-the-docs/just-the-docs/issues/982.
+@include mq(lg) {
+    .side-bar {
+        min-width: 400px;
+    }
+
+    .site-nav, .site-header {
+        width: 400px;
+    }
+}
+
+// Add support for external anchor icons.
+.external > svg {
+    width: 1rem;
+    vertical-align: text-bottom;
+}
```

Comment:
This is used for the external anchor icons, when you look at the "view source code" button.

You are right that that functionality got stripped out of this PR, as that's a follow-up in `gdejong/docgen`. I can remove if you'd prefer.

Reply:
Ah, check! If it's not too much work: let's (since officially the change doesn't belong here). But since we do plan to work on `gdejong/docgen`: no strong objection to keep it.

Reply:
Fair point. Let's keep it clean.